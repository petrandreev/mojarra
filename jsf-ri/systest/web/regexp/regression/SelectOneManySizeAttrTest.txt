(?s).*<html>\s*<head>\s*<title>\s*Select.One,Many.ListBox\s*Size\s*Attribute\s*Test\s*</title>\s*</head>\s*<body>\s*<select\s*name=".*"\s*size="5">\s*<option\s*value="val1">\s*val1\s*</option>\s*</select>\s*<select\s*name=".*"\s*multiple="multiple"\s*size="5">\s*<option\s*value="val1">\s*val1\s*</option>\s*</select>\s*</body>\s*</html>.*
