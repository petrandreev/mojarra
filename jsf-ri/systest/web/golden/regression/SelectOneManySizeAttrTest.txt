







<html>
    <head>
        <title>Select{One,Many}ListBox Size Attribute Test</title>        
    </head>
    <body>
        <select id="j_id_id16" name="j_id_id16" size="5">	<option value="val1">val1</option>
</select><select id="j_id_id21" name="j_id_id21" multiple="multiple" size="5">	<option value="val1">val1</option>
</select>
    </body>
</html>
